### PR TITLE
firewalls: Add `nftables` subsection.

### LINKED ISSUE #266: iptables and nftables
Nftables is currently completely missing.

The current iptables documentation seems low quality imho.

* The section [IP6 firewall rules](https://docs.voidlinux.org/config/network/iptables.html#ip6-firewall-rules) could be merged with "Applying the rules at boot", the `iptables-runit` service works already for both, the snipped for `/etc/rc.local` could do exactly the same.
* [Firewall rules](https://docs.voidlinux.org/config/network/iptables.html#firewall-rules) and its subsection [Adjusting the rules](https://docs.voidlinux.org/config/network/iptables.html#adjusting-the-rules) explain how to edit files, which shouldn't be necessary. They also miss to document the file names of the rules which would be loaded on boot with the `rc.local` snipped, the runit "service" and the `iptables-runit` core service.

### PR DESCRIPTION
**Pull request by flexibeast**
This PR will close #266.

### CODE REVIEW

File: src/config/network/firewalls.md [hunk 1]
```diff
@@ -53,3 +53,38 @@ and restore them by upping the relevant service, e.g.:
...
 # sv up ip6tables
 ```
+
+## nftables
+
+`nftables` replaces `iptables`, `ip6tables`, `arptables` and `ebtables`
+(collectively referred to as `xtables`). The [nftables
+wiki](https://wiki.nftables.org/wiki-nftables/index.php/Main_Page) describes
+[the main
+differences](https://wiki.nftables.org/wiki-nftables/index.php/Main_differences_with_iptables).
```
Comment (ericonr) [inline src/config/network/firewalls.md:63]: "the main differences from those other tools" or "other technologies", perhaps?

File: src/config/network/firewalls.md [hunk 2]
```diff
@@ -53,3 +53,38 @@ and restore them by upping the relevant service, e.g.:
...
+
+To use `nftables`, install the `nftables` package, which provides
+[nft(8)](https://man.voidlinux.org/nft.8). It also provides
+[iptables-translate(8)/ip6tables-translate](https://man.voidlinux.org/iptables-translate.8)
```
Comment (ericonr) [inline src/config/network/firewalls.md:67]: ip6tables-translate(8)

File: src/config/network/firewalls.md [hunk 3]
```diff
@@ -53,3 +53,38 @@ and restore them by upping the relevant service, e.g.:
...
+and
+[iptables-restore-translate(8)/ip6tables-restore-translate(8)](https://man.voidlinux.org/iptables-restore-translate.8),
+which convert `iptables` rules to `nftables` rules.
+
+### Applying the rules at boot
+
+To apply nttables rules at runit stage 1, install the `runit-nftables` package.
```
Comment (ericonr) [inline src/config/network/firewalls.md:74]: nftables

File: src/config/network/firewalls.md [hunk 4]
```diff
@@ -53,3 +53,38 @@ and restore them by upping the relevant service, e.g.:
...
+This adds a core-service which restores the ruleset in `/etc/nftables.conf`.
+
+### Applying the rules at runtime
+
+The `nftables` provides the `nftables` service to load rules from
```
Comment (ericonr) [inline src/config/network/firewalls.md:79]: The `nftables` package

File: src/config/network/firewalls.md [hunk 5]
```diff
@@ -53,3 +53,38 @@ and restore them by upping the relevant service, e.g.:
...
+
+```
+# sv up nftables
+```
+
+and flush them:
+
+```
+# sv down nftables
+```
```
Comment (ericonr) [inline src/config/network/firewalls.md:90]: For this one I would suggest something like:

"The nftables package provides the nftables service, which uses rules from `/etc/nftables.conf`. To load them, run the command below:"

**Comment by flexibeast** [comment]
@ericonr: Thanks for the review. i've fixed the typos and made some changes; let me know if you think i can now merge this.

(CI failing due to zeroconf.org currently being inaccessible.)

File: src/config/network/firewalls.md [hunk 6]
```diff
@@ -53,3 +53,39 @@ and restore them by upping the relevant service, e.g.:
...
+
+### Applying the rules at boot
+
+To apply nftables rules at runit stage 1, install the `runit-nftables` package.
+This adds a core-service which restores the ruleset in `/etc/nftables.conf`.
+
+### Applying the rules at runtime
+
+The `nftables` package provides the `nftables` service, which uses rules from
+`/etc/nftables.conf`. To load the rules, run:"
```
Comment (ericonr) [inline src/config/network/firewalls.md:81]: Leftover quote.

**Pull request merged**